biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -113,8 +113,7 @@ public Jar build() throws Exception {
 			File mff = getFile(mf);
 			if (mff.isFile()) {
 				updateModified(mff.lastModified(), "Manifest " + mff);
-				try {
-					InputStream in = new FileInputStream(mff);
+				try (InputStream in = new FileInputStream(mff)){
 					manifest = new Manifest(in);
 					in.close();
```

Comment:
There is already a close here. If you want to use try-with-resource, then remove the explicit close call.